src/github.com/vharitonsky/iniflags/.gitignore

```diff
@@ -0,0 +1 @@
+tags
```

Comment:
Actually, that's only needed by the main package, which we're not using here, so I can remove it.  Thanks!